deployment/terraform/app.tf

```diff
@@ -22,6 +22,10 @@ resource "aws_cloudwatch_log_group" "potsdam" {
   }
 }
 
+data "aws_iam_role" "autoscaling" {
```

Comment:
Did you consider bumping the `terraform-aws-ecs-web-service` version to 0.4.0 here? That should remove the need to supply `ecs_autoscale_role_arn`.

Reply:
I did; the changes made to `step_scaling_policy` in  azavea/terraform-aws-ecs-web-service/pull/7 are incompatible with older versions of the AWS provider.

```
9 error(s) occurred:

* module.potsdam_ecs_service.aws_appautoscaling_policy.down: "adjustment_type": required field is not set
* module.potsdam_ecs_service.aws_appautoscaling_policy.down: "cooldown": required field is not set
* module.potsdam_ecs_service.aws_appautoscaling_policy.down: "metric_aggregation_type": required field is not set
* module.potsdam_ecs_service.aws_appautoscaling_policy.down: : invalid or unknown key: step_scaling_policy_configuration
* module.potsdam_ecs_service.aws_appautoscaling_policy.up: "adjustment_type": required field is not set
* module.potsdam_ecs_service.aws_appautoscaling_policy.up: "cooldown": required field is not set
* module.potsdam_ecs_service.aws_appautoscaling_policy.up: "metric_aggregation_type": required field is not set
* module.potsdam_ecs_service.aws_appautoscaling_policy.up: : invalid or unknown key: step_scaling_policy_configuration
* module.potsdam_ecs_service.aws_appautoscaling_target.main: "role_arn": required field is not set
```